canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -276,13 +279,11 @@ class Toolbar extends React.Component {
 		this.leftBar = this.props.config.leftBar || [];
 		this.rightBar = this.props.config.rightBar || [];
 		this.rightBar = [...this.rightBar].reverse() || [];
-
 		const leftItems = this.generateToolbarItems(this.leftBar, false, true);
 		const rightItems = this.generateToolbarItems(this.rightBar, false, false);
-
 		const canvasToolbar = (
 			<ReactResizeDetector handleWidth onResize={this.onToolbarResize}>
-				<div className="toolbar-div" instanceid={this.props.instanceId}>
+				<div className="toolbar-div" instanceid={this.props.instanceId} >
```

Comment:
Why has a space been inserted in this line?  Please don't change lines of code unless they are part of your code changes. You should be reviewing your code before asking me to review it and making sure stuff like this isn't left in. That will save me having to write comments like this.

Reply:
Adjusted it.